File: javascript/array-of-objects/exercise01/classCar.js
class Car {
  constructor(make, model, color) {
    this.make = make;
    this.model = model;
    this.color = color;
  }

  // Create method to display car details
  drive() {
    console.log(`You are driving a ${this.color} ${this.make} ${this.model}`);
  }
}

module.exports = Car;

File: python/array-of-objects/exercise01/classCar.py
class Car:
  def __init__(self, make, model, color):
    self.make = make
    self.model = model
    self.color = color

# Create a method to display the car details
  def drive(self):
    print(f"You are driving a {self.color} {self.make} {self.model}")